cicd/csm_agent.spec

```diff
@@ -76,14 +74,12 @@ systemctl stop csm_agent
 
 %postun
 [ $1 -eq 1 ] && exit 0
-rm -f /etc/systemd/system/csm_agent.service 2> /dev/null;
```

Comment:
Removal of the below lines will change the way csm agent rpm is uninstalled in LR environment - 
rm -f /etc/systemd/system/csm_agent.service 2> /dev/null;
systemctl daemon-reload

Instead of removing these lines we will only execute the above commands if /etc/systemd/system/csm_agent.service file is present.

Similarly we can do the same for %preun scriptlet.

Reply:
Fixed.